src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -949,6 +949,24 @@ trait BronzeTransforms extends SparkSessionWrapper {
 
   }
 
+  /**
+   *
+   * @param cloudProvider
+   * @param inputCol
+   * @param isMultiWorkSpaceDeployment
+   * @param organisationId
+   * @return
+   */
+  private def fetchClusterLogConfiguration(cloudProvider: String, inputCol: Column,
+                                   isMultiWorkSpaceDeployment: Boolean, organisationId: String): Column = {
+    if(cloudProvider.toLowerCase() == "gcp" && isMultiWorkSpaceDeployment)
```

Comment:
add curly braces around if the if logic and remove the old comment lines.

Reply:
done